dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/configMLDT/configForEutaxonomyFinancialsMLDT.ts

```diff
@@ -71,42 +82,43 @@ export const configForEuTaxonomyFinancialsMLDT = [
         type: "cell",
         label: euTaxonomyKpiNameMappings.fiscalYearDeviation,
         explanation: euTaxonomyKpiInfoMappings.fiscalYearDeviation,
-        shouldDisplay: (): boolean => true,
+        shouldDisplay: (dataset: EuTaxonomyDataForFinancials): boolean => !!dataset.fiscalYearDeviation,
```

Comment:
why not use `setShowIf` here as well? why make a difference for exactly these values?

Reply:
same as other comment on issue